grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientOptions.java

```diff
@@ -165,5 +166,12 @@ public final class GrpcClientOptions {
     public static final ClientOption<CallCredentials> CALL_CREDENTIALS =
             ClientOption.define("GRPC_CLIENT_CALL_CREDENTIALS", NullCallCredentials.INSTANCE);
 
+    /**
+     * Sets the {@link CallCredentials} that carries credential data that will be propagated to the server
+     * via request metadata.
+     */
+    public static final ClientOption<GrpcExceptionHandlerFunction> EXCEPTION_HANDLER =
+            ClientOption.define("EXCEPTION_HANDLER", (ctx, cause, metadata) -> null);
```

Comment:
Sorry, I don't think I understood your intention 😅 

Do you mean you prefer that `(ctx, cause, metadata) -> null` is defined as the "DEFAULT" exception handler somewhere? (so that `GrpcExceptionHandlerFunction` is non-null overall?)

Reply:
I guess `(ctx, cause, metadata) -> null` is the default exception handler which returns null so that the status from `statusFromThrowable` is used.
https://github.com/line/armeria/blob/d30db0c88d178468c3bb03a4e1d920b4c7a897b9/grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatus.java#L115-L122
I think we can explicitly call `(ctx, cause, metadata) -> GrpcStatus.fromThrowable(cause)` or leave a comment about why null is returned to make it clearer. 🤔

Reply:
Updated as suggested. Note that we still need the following fallback logic since `ClientOption` can be overridden

https://github.com/line/armeria/blob/d30db0c88d178468c3bb03a4e1d920b4c7a897b9/grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatus.java#L122